### PR TITLE
remove more px4 remnants

### PR DESCRIPTION
**Pull request by tridge**
small bits throughout the tree

### CODE REVIEW

File: libraries/AP_Relay/AP_Relay.cpp [hunk 1]
```diff
@@ -50,28 +50,28 @@ const AP_Param::GroupInfo AP_Relay::var_info[] = {
     // @DisplayName: First Relay Pin
     // @Description: Digital pin number for first relay control. This is the pin used for camera control.
     // @User: Standard
-    // @Values: -1:Disabled,49:BB Blue GP0 pin 4,50:Pixhawk AUXOUT1,51:Pixhawk AUXOUT2,52:Pixhawk AUXOUT3,53:Pixhawk AUXOUT4,54:Pixhawk AUXOUT5,55:Pixhawk AUXOUT6,57:BB Blue GP0 pin 3,111:PX4 FMU Relay1,112:PX4 FMU Relay2,113:PX4IO Relay1/BB Blue GP0 pin 6,114:PX4IO Relay2,115:PX4IO ACC1,116:PX4IO ACC2/BB Blue GP0 pin 5
+    // @Values: -1:Disabled,49:BB Blue GP0 pin 4,50:AUXOUT1,51:AUXOUT2,52:AUXOUT3,53:AUXOUT4,54:AUXOUT5,55:AUXOUT6,57:BB Blue GP0 pin 3,113:BB Blue GP0 pin 6,115:PX4IO ACC1.116:BB Blue GP0 pin 5
```
Comment (rmackay9) [inline libraries/AP_Relay/AP_Relay.cpp]: could also rename "PX4IO ACC1" to just "IO ACC1"

**Comment by rmackay9** [comment]
It's a little outside my area but it looks OK to me.

I see the VibeTest example sketch was removed which seems a bit unrelated but perhaps it's just not being maintained updated?

File: libraries/AP_Relay/AP_Relay.cpp (hunk 1 above)
Comment (tridge) [reply libraries/AP_Relay/AP_Relay.cpp]: i've removed it, its unused

**Comment by tridge** [comment]
vibtest code was px4 specific

**Pull request merged**